metaflow/plugins/aip/tests/test_argo_support.py

```diff
@@ -8,7 +8,7 @@
 
 
 disabled_test_flows = [
-    "kfp_flow.py",  # kfp_preceding_component feature has been deprecated.
+    "aip_flow.py",  # kfp_preceding_component feature has been deprecated.
```

Comment:
We should be ready to completely remove kfp_preceding_component now, right?

Reply:
I kept it because adding this feature was very complicated, and re-adding it would be too.
Instead we are:
* not exposing it to customers
* we'll remove by migrating to Metaflow Argo plugin, or by refactoring WFSDK AIP plugin to not use kfp compiler.  I added our next steps as a subject to discuss in our offsite.